activesupport/lib/active_support/dependencies.rb

```diff
@@ -760,7 +736,6 @@ def remove_constant(const) #:nodoc:
         begin
           constantized = parent.const_get(to_remove, false)
         rescue NameError
-          log "the constant #{const} is not reachable anymore, skipping"
```

Comment:
@matthewd Thanks for your advice. I spend some time myself before reading your comment and I did the same thing :grimacing: